aten/src/ATen/cuda/tunable/Tunable.cpp

```diff
@@ -107,6 +107,34 @@ void TuningResultsManager::Add(const std::string& op_signature, const std::strin
   AddImpl(op_signature, params_signature, best, it->second);
 }
 
+void TuningResultsManager::RecordUntuned( std::ofstream& untuned_file, const std::string& op_signature, const std::string& params_signature) {
+  std::scoped_lock l{lock_};
+  if (!untuned_file.good()){
+    TORCH_WARN("failed to open file for writing; untuned gemm will not be saved");
```

Comment:
```suggestion
    TORCH_WARN_ONCE("failed to open file for writing; untuned gemm will not be saved");
```
otherwise if the file cannot be opened for writing, this will flood the screen with this message.